Update for SEO and point to new docs (#721)
MicrosoftDocs/WindowsCommunityToolkitDocs: docs/index.md

author: nmetulev
ms.topic: overview
description: The Windows Community Toolkit is a collection of helper functions, custom controls, and app services. It simplifies and demonstrates common developer tasks building UWP apps for Windows 10. 
---

# Windows Community Toolkit Documentation

> [!IMPORTANT]

> > This Windows Community Toolkit is moving to an archived state. See the updated [Windows Community Toolkit documentation](/dotnet/communitytoolkit/windows/) in the .NET guidance.

Reviewer: @mattwojo I would maybe change this to "This Windows Community Toolkit **documentation** is moving to an archived state." I don't know if you want to make statements about the status of the toolkit itself. But I don't know much about it, so just a thought.

Author: Great catch! Updated.

> > This Windows Community Toolkit documentation is moving to an archived state. See the updated [Windows Community Toolkit documentation](/dotnet/communitytoolkit/windows/) in the .NET guidance.

Reviewer 2: ```suggestion
> This Windows Community Toolkit documentation has moved! See the updated [Windows Community Toolkit documentation](/dotnet/communitytoolkit/windows/) in the .NET guidance.
```
